fiftyone/core/annotation/generate_label_schema.py

```
@@ -0,0 +1,563 @@
+            f.name == foac.BOUNDING_BOX
+            and field.document_type == fol.Detection
+        ):
+            # bounding_box is a list of floats field, but really a 4-tuple of
+            # [0, 1] floats, omit for special handling by the App
+            continue
+
+        try:
+            attributes[f.name] = _generate_field_label_schema(
+                collection, f"{field_name}.{f.name}", scan_samples
+            )
+        except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)